jekyll-docs/datastores.md

```diff
@@ -54,3 +54,11 @@ vsphere			defaultDsVolume
 vsphere			ds1volume@datastore1
 photon			ds2volume@datastore2
 ```
+
+Datastore decision is taking place as shown below (preference from top to bottom)
+
+1. Using volume long/fully qualified name (with @<datastore_name>): volume is created on custom datastore
+2. Using short name (not passed @<datastore_name>)
+   2.1 vmgroup's default_datastore is chosen if set
+   2.2 docker_host vm's datastore is chosen as default if none of above condition met.
```

Comment:
This should be changed to:
2. Using short name (not passed @<datastore_name>): volume is created on the vmgroup's default_datastore.